src/main/java/com/microsoft/azure/functions/worker/reflect/DefaultClassLoaderProvider.java

```diff
@@ -30,18 +30,23 @@ public ClassLoader createClassLoader() {
     return classLoader;
   }
 
-  @Override
-  public void addUrl(URL url) throws IOException {
+  public void addUrl(URL url, String source) throws IOException {
```

Comment:
```suggestion
  public void addUrl(URL url) throws IOException {
```
Please let us check if source is necessary

Reply:
```suggestion
  private void addUrl(URL url, String source) throws IOException {
```